source/auth/auth.md

```diff
@@ -1480,7 +1479,7 @@ An example human callback API might look like:
 ```typescript
 interface IdpInfo {
   issuer: string;
-  clientId: string;
```

Comment:
What is the intended behavior when `clientId` is `None`? The code I have written on client side assumes the `clientId` will always exist.

Reply:
It would be None if the user used a Human Callback and the IdP is configured on the server with `supportsHumanFlows:False`.  They shouldn't really be doing that, but it is allowed.  See https://jira.mongodb.org/browse/DRIVERS-2773 for details.

Reply:
We may need a test to cover this case